MVCDemos/Content/DemosStyles/TreeList/Adaptability.css

```diff
@@ -1,3 +1,4 @@
 #employees {
     padding: 10px;
+    max-height: 440px;
 }
```

Comment:
Let's fix the ```end of file``` here :)

Reply:
It's linter issue, let's do it in separate card for all files